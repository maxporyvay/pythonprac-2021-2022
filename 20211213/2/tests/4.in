[11, 63, 91, 30, 11, 28, 28, 82, 31, 68, 64, 40, 64, 15, 83, 3, 44, -7, 78, 66, 38, 18, 66, 29, -10, 3, 28, 80, 68, 9, 72, 12, -1, 40, 78, 49, 83, 28, 92, 50, 48, -4, 78, 54, 27, -10, 55, -8]