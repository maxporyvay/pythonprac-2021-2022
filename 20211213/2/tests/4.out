[-10, -10, -8, -7, -4, -1, 3, 3, 9, 11, 11, 12, 15, 18, 27, 28, 28, 28, 28, 29, 30, 31, 38, 40, 40, 44, 48, 49, 50, 54, 55, 63, 64, 64, 66, 66, 68, 68, 72, 78, 78, 78, 80, 82, 83, 83, 91, 92]
